### DISCUSSION TITLE
打算读这本英文小说  Burma Sahib

### DISCUSSION THREAD

**Discussion by megtog** [IntensiveCoLearning/english#70] [Q&A]
哈喽，感觉自己做不到天天学习，所以没报名，但对英语学习还是蛮感兴趣的

刚听了这个播客

<https://sinica.substack.com/p/peter-hessler-on-his-new-book-other>

最后的部分 Peter 推荐了小说 Burma Sahib

学语言到能读小说应该是比较难的，想试试能不能读进去

**Comment by luffythink** [comment]
通过阅读英语原著建立起学习英语的兴趣，是一个很好的点呀，期待你之后读进去后的反馈与交流。
